drawBot/context/tools/pilTools.py

```diff
@@ -0,0 +1,11 @@
+import AppKit
+import io
+
+
+def pilToNSImage(pilImage):
+    buffer = io.BytesIO()
+    pilImage.save(buffer, pilImage.format)
```

Comment:
Why keep/use `pilImage.format`? What if the original file was in a format that PIL can read, but NSImage cannot? I'd suggest to use PNG here. There's for example no need to recompress to JPG, as NSImage will have to unpack into a pixel array anyway.

Reply:
+1 for PNG

Reply:
now internally the image objects are build with from the TIFF representation

Reply:
see https://github.com/typemytype/drawbot/pull/452/files#diff-7c0b8578fa67d5d3f0a2ea4b08902cf57175c06b1b8fd6b3cf0c275d804417faR13

Reply:
an maybe `pilToNSImage`  has to move to `drawBot.misc` which is a bit more accessible then `drawBot.context.tools.pilTools`